modules/cxf/pom.xml

```diff
@@ -667,6 +667,13 @@
 
         <!-- Unit tests -->
         <dependency>
+            <groupId>org.mule</groupId>
```

Comment:
I think the client module needs this dependency too doesn't it? Anything that uses the test transport right?